booosta_proto/proto/personnel.proto

```diff
@@ -5,6 +5,7 @@ service PersonnelService {
     rpc GetRetailerByUserId(GetRetailerByUserIdRequest) returns (GetRetailerByUserIdResponse) {}
     rpc GetAgentByUserId(GetAgentByUserIdRequest) returns (GetAgentByUserIdResponse) {}
     rpc GetAggregatorByUserId(GetAggregatorByUserIdRequest) returns (GetAggregatorByUserIdResponse) {}
+    rpc GetDeviceTerminalIdByRetailerId(GetDeviceTerminalIdByRetailerIdRequest) returns (GetDeviceTerminalIdByRetailerIdResponse) {}
```

Comment:
Can you change this to user_id, we are actually using the user id not the retailer id